markdown-link-check

```diff
@@ -303,7 +303,7 @@ async function runMarkdownLinkCheck(filenameForOutput, markdown, opts) {
     ));
 
     if (err) throw null;
-    else if (results.some((result) => result.status === 'dead')) return;
+    else if (results.some((result) => result.status === 'dead')) throw null;
```

Comment:
I have enabled the workflows. I'm hoping a test will fail as a result of this change. If not, we should add one.

Reply:
OK. I'll try to add some test case

Reply:
Added test cases https://github.com/tcort/markdown-link-check/pull/452/commits/dc70b823fcddc6cc555f741f0a657487c214fb13